bci_tester/data.py

```diff
@@ -664,6 +664,21 @@ def create_BCI(
     for mariadb_client_ver, os_versions in _MARIADB_VERSION_OS_MATRIX
 ]
 
+POSTFIX_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/postfix:{postfix_ver}",
```

Comment:
this needs to be now adjusted after the switch to appcol:

```suggestion
        build_tag=f"{SAC_CONTAINER_PREFIX}/postfix:{postfix_ver}",
```